pkg/storage/export/export/vm-source_test.go

```diff
@@ -408,6 +411,34 @@ var _ = Describe("PVC source", func() {
 		Entry("PVCs", createVMWithPVCs, "kubevirt", "kubevirt", verifyKubevirtInternal),
 		Entry("Memorydump and pvc", createVMWithPVCandMemoryDump, "kubevirt", "archive", verifyMixedInternal),
 	)
+	It("Should create VM export, when VM is stopped, but VMI exists in succeeded state", func() {
```

Comment:
Does it make sense to add also a test for the failed case?

Reply:
yeah I agree should be a pretty small change to make it test also the failed cased. other then that looks good and I will lgtm

Reply:
Done